tst/EnergyPlus/unit/Fixtures/GroundTempsFixture.hh

```diff
@@ -0,0 +1,39 @@
+#ifndef GroundTempsFixture_hh_INCLUDED
+#define GroundTempsFixture_hh_INCLUDED
+
+// Google test headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include "EnergyPlusFixture.hh"
+#include <EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.hh>
+#include <EnergyPlus/GroundTemperatureModeling/GroundTemperatureModelManager.hh>
+#include <EnergyPlus/GroundTemperatureModeling/KusudaAchenbachGroundTemperatureModel.hh>
+#include <EnergyPlus/GroundTemperatureModeling/SiteBuildingSurfaceGroundTemperatures.hh>
+#include <EnergyPlus/GroundTemperatureModeling/SiteDeepGroundTemperatures.hh>
+#include <EnergyPlus/GroundTemperatureModeling/SiteFCFactorMethodGroundTemperatures.hh>
+#include <EnergyPlus/GroundTemperatureModeling/SiteShallowGroundTemperatures.hh>
+#include <EnergyPlus/GroundTemperatureModeling/XingGroundTemperatureModel.hh>
+#include <EnergyPlus/UtilityRoutines.hh>
+
+namespace EnergyPlus {
+
+	class GroundTempsFixture : public EnergyPlusFixture
+	{
+	protected:
+		static void SetUpTestCase() { }
+		static void TearDownTestCase() { }
+		
+		virtual void SetUp() {
+			EnergyPlusFixture::SetUp();  // Sets up the base fixture first.       
+		}
+
+		virtual void TearDown() {
+			GroundTemperatureManager::clear_state();		
```

Comment:
@mmatts I would lean towards removing this fixture and adding this clear_state() in EnergyPlusFixture instead. Then use that fixture in your tests. Also you only need to include GroundTemperatureModelManager.hh in the fixture and then in the unit test file you can include the other header files.